### PR TITLE
update on the collapse

### PR DESCRIPTION
**Pull request by abdulvasay-234**
the design looks good if it is in bold like other headings

### CODE REVIEW

**Review by abdulvasay-234 — COMMENTED** [review]
its good to go

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ format that cannot be directly used for machine learning models.
 machine learning model which also increases the accuracy and efficiency of a machine 
 learning model.
 <details>
-  <summary>Steps involved in Data pre-processing:</summary>
+  <summary><h2>Steps involved in Data pre-processing:</h2></summary>
```
Comment (afnanabdulvasay-234) [inline README.md:16]: how about adding the same text as used previous. (using H1 tag)

**Review by afnanabdulvasay-234 — COMMENTED** [review]
i think its good

**Comment by afnanabdulvasay-234** [comment]
how about h1 tag

**Pull request merged**